src/lib/components/Widget.js

```diff
@@ -112,6 +112,7 @@ export const Widget = React.forwardRef((props, forwardedRef) => {
     setElement(null);
     if (!code) {
       if (code === undefined) {
+        near.config.errorCallback({scope: "Source code not found", message: src});
```

Comment:
This PR makes total sense to me, but I feel that this implementation could be improved by having all the possible variants of scopes explicitly defined in one place (enum), otherwise one would need to grep through the whole codebase to see all possible contexts that needs to be handled